src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/EventSummaryCreator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import java.util.List;
+
+public class EventSummaryCreator {
+
+    private static final String EVENT_SUMMARY_REPLACE = "{XX}";
+    private static final String EVENT_SUMMARY_TRIM_DOTS = "...";
+    private static String EVENT_SUMMARY_TEMP =
+        "Attaching exception record(%d) document numbers:" + EVENT_SUMMARY_REPLACE + " to case:%d";
+    private static int EVENT_SUMMARY_MAX_SIZE = 1024;
```

Comment:
Do we know this value?

Reply:
it is ccd restriction it is in the incident.